developer/tutorial/customization.md

```diff
@@ -366,15 +358,6 @@ specified which template we would use for a "notFound". When we get to the routi
 One important thing to understand is that at any point in time when RC goes to render a route/page it's going to
 determine how to pull the layout record from a key of `layout + workflow`. The `coreWorkflow` is a special case in that it is a workflow with just one step. It is essentially the "default" workflow when you hit the home page.
 
-Now to make this file active, we need to add an import to the `main.js` in the root `server` directory.
-
-```
-/**
- * Custom Plugins
- */
-import "/imports/plugins/custom/beesknees/register";
-```
-
 Because we have made changes to the registry, you must now `./reaction reset` for your changes to take affect.
```

Comment:
If a register.js doesn't exist on next run/build, Reaction removes it from the registry.  That's the expected behavior, right?